bin/latex.py

```diff
@@ -144,20 +141,50 @@ def get_environment():
     return env
 
 
+def build_latex_pdf(builddir, tex_path, problem_path=None):
+    env = get_environment()
+    run_once = getattr(config.args, '1', False)
+    watch = getattr(config.args, 'watch', False)
+    ret = util.exec_command(
+        ['latexmk', '-cd', '-g', '-pdf',
+         '-pdflatex=pdflatex -interaction=nonstopmode -halt-on-error']
+        + (['-pvc'] if watch else [])
```

Comment:
maybe refactor the command to a temp variable and do `if watch: command.append('-pvc')`.
(But if it doesn't look cleaner in the end; don't bother.)

Reply:
It looks slightly more Pythonic with your proposed refactoring, thanks for the suggestion! :slightly_smiling_face: